resources/webidl2/build.sh

```diff
@@ -0,0 +1,12 @@
+set -ex
+
+if [ ! -d "webidl2.js" ]; then
+  git clone git@github.com:w3c/webidl2.js.git
+fi
+cd webidl2.js
+npm run build-debug
+VERSION=$(jq -r ".version" < package.json)
```

Comment:
Unless every commit is released, this version will be misleading, not the same thing that got published as that version on npm.

I would suggest either installing from npm (similar to the `@webref/idl` workflow) or just using the commit and not recording the version.

Reply:
Fair enough. I went with not recording the version, since installing from npm seems to give minified code